test/higher_order_ops/test_invoke_subgraph.py

```diff
@@ -1481,6 +1481,119 @@ def fn(x):
+            cos: "f32[s48, 16]" = torch.ops.aten.cos.default(primals_1)
+            return (cos, primals_1, primals_0)
+
+    class ___forward_invoke_subgraph_1_0_post_graph(torch.nn.Module):
+        def forward(self, primals_0: "Sym(s48)", primals_1: "f32[s48, 16]"):
+            cos: "f32[s48, 16]" = torch.ops.aten.cos.default(primals_1)
+            return (cos, primals_1, primals_0)
+""",
+            )
+            self.assertExpectedInline(
+                normalize_gm(backend.bw_graphs[0].print_readable(print_output=False)),
+                """\
+class GraphModule(torch.nn.Module):
+    def forward(self, primals_1: "Sym(s48)", getitem_9: "Sym(s48)", getitem_11: "Sym(s48)", getitem_8: "f32[s48, 16]", getitem_10: "f32[s48, 16]", cos: "f32[s48, 16]", tangents_1: "f32[]"):
+        expand: "f32[s48, 16]" = torch.ops.aten.expand.default(tangents_1, [primals_1, 16]);  tangents_1 = primals_1 = None
+
+        ___backward_invoke_subgraph_1_0_post_graph = self.___backward_invoke_subgraph_1_0_post_graph
+
+        invoke_subgraph_7 = torch.ops.higher_order.invoke_subgraph(___backward_invoke_subgraph_1_0_post_graph, '___backward_invoke_subgraph_1_0_post_graph', (getitem_11, getitem_10, expand));  ___backward_invoke_subgraph_1_0_post_graph = getitem_11 = getitem_10 = expand = None
+        getitem_3: "f32[s48, 16]" = invoke_subgraph_7[1];  invoke_subgraph_7 = None
+
+        mul_6: "f32[s48, 16]" = torch.ops.aten.mul.Tensor(getitem_3, cos);  getitem_3 = cos = None
+
+        ___backward_invoke_subgraph_0_0_post_graph = self.___backward_invoke_subgraph_0_0_post_graph
+
+        invoke_subgraph_5 = torch.ops.higher_order.invoke_subgraph(___backward_invoke_subgraph_0_0_post_graph, '___backward_invoke_subgraph_0_0_post_graph', (getitem_9, getitem_8, mul_6));  ___backward_invoke_subgraph_0_0_post_graph = getitem_9 = getitem_8 = mul_6 = None
+        getitem_6: "f32[s48, 16]" = invoke_subgraph_5[1];  invoke_subgraph_5 = None
+        return (None, getitem_6)
+
+    class ___backward_invoke_subgraph_1_0_post_graph(torch.nn.Module):
+        def forward(self, primals_0: "Sym(s48)", primals_1: "f32[s48, 16]", tangents_0: "f32[s48, 16]"):
+            sin: "f32[s48, 16]" = torch.ops.aten.sin.default(primals_1);  primals_1 = None
+            neg: "f32[s48, 16]" = torch.ops.aten.neg.default(sin);  sin = None
+            mul_9: "f32[s48, 16]" = torch.ops.aten.mul.Tensor(tangents_0, neg);  tangents_0 = neg = None
+            return (None, mul_9)
+
+    class ___backward_invoke_subgraph_0_0_post_graph(torch.nn.Module):
+        def forward(self, primals_0: "Sym(s48)", primals_1: "f32[s48, 16]", tangents_0: "f32[s48, 16]"):
+            sin: "f32[s48, 16]" = torch.ops.aten.sin.default(primals_1);  primals_1 = None
+            neg: "f32[s48, 16]" = torch.ops.aten.neg.default(sin);  sin = None
+            mul_10: "f32[s48, 16]" = torch.ops.aten.mul.Tensor(tangents_0, neg);  tangents_0 = neg = None
+            return (None, mul_10)
+""",
+            )
```

Comment:
Why do the graphs have the same (contiguous) strides for the tangents? Is that expected?

Reply:
Strides are not printed for some reason in this expected inline test. When I look at tlp, the strides are different. These are just sizes.

Reply:
Oh oops I misinterpreted